functions/adjustChatCapacity.ts

```diff
@@ -120,6 +121,17 @@ export const adjustChatCapacity = async (
     return { status: 200, message: 'Successfully decreased channel capacity' };
   }
 
+  if (body.adjustment === 'setTo1') {
+    if (channel.configuredCapacity !== 1) {
+      await channel.update({ capacity: channel.configuredCapacity - 1 });
```

Comment:
When I see a condition named `setTo1`, I was expecting the update code to be:
`await channel.update({ capacity: 1 });`

Is there a reason we're doing:
`await channel.update({ capacity: channel.configuredCapacity - 1 });`

Reply:
No, good catch